runtime/compiler/control/CompilationThread.cpp

```diff
@@ -4406,7 +4415,11 @@ TR::CompilationInfoPerThread::processEntry(TR_MethodToBeCompiled &entry, J9::J9S
             compInfo->getQueueWeight(),
             compInfo->getNumCompThreadsActive(),
             compInfo->getRampDownMCT() ? "RampDownMCT" : "",
-            compInfo->getSuspendThreadDueToLowPhysicalMemory() ? "LowPhysicalMem" : "");
+            compInfo->getSuspendThreadDueToLowPhysicalMemory() ? "LowPhysicalMem" : "",
+#if defined(J9VM_OPT_JITSERVER)
+            compInfo->serverHasLowPhysicalMemory() ? "ServerLowPhysicalMem" : ""
```

Comment:
TR_VerboseLog::writeLineLocked needs another %s for the new string. It gets tricky though due to the ifdef. I am ok with being a little imprecise and say just "LowPhysicalMem", but the decision should come from either the local JVM and or the remote server:
```
bool lowPhysicalMem = compInfo->getSuspendThreadDueToLowPhysicalMemory();
#if defined(J9VM_OPT_JITSERVER)
lowPhysicalMem = lowPhysicalMem || compInfo->serverHasLowPhysicalMemory();
#endif

Reply:
Yeah, or we could print an empty string if JITServer is not enabled:
```
   compInfo->getRampDownMCT() ? "RampDownMCT" : "",
   compInfo->getSuspendThreadDueToLowPhysicalMemory() ? "LowPhysicalMem" : "",
#if defined(J9VM_OPT_JITSERVER)
   compInfo->serverHasLowPhysicalMemory() ? "ServerLowPhysicalMem" :
#endif
    ""
```

Reply:
I am fine either way